shapely/predicates.py

```diff
@@ -501,6 +503,7 @@ def contains(a, b, **kwargs):
     within : ``contains(A, B) == within(B, A)``
     contains_properly : contains with no common boundary points
     prepare : improve performance by preparing ``a`` (the first argument)
+    contains_xy : variant for checking against a Point with x, y coordinates
```

Comment:
I believe this function is currently not there (only in the `.lib.`). Which is (I think) indeed the outcome of the decision.

Reply:
Yeah, that's because in the last commit I just tested with what it would look like to have it embedded in the single `contains()` function instead of a public `contains_xy()`, but without updating all docs (before deciding what we actually want, see https://github.com/shapely/shapely/pull/1548#issuecomment-1311746904)